test/ng/directive/selectSpec.js

```diff
@@ -611,6 +622,24 @@ describe('select', function() {
 
     });
 
+
+    it('should not break when adding options via a directive with `replace: true` '
+        + 'and a structural directive in its template',
+      function() {
+        scope.options = [
+          {value: '1', label: 'Option 1'},
+          {value: '2', label: 'Option 2'},
+          {value: '3', label: 'Option 3'}
+        ];
+
+        expect(function() {
```

Comment:
I would prefer that we check the validity of the select itself: `expect(element).toEqualSelect([unknownValue()], '1', '2', '3');`